packages/storefront/src/pages/license.md

```diff
@@ -6,36 +6,9 @@ reserves the right to change and update this licensing agreement at any time.
 
 <TableOfContents></TableOfContents>
 
-## Restrictions
+## Agreement
 
-Porsche AG grants you a limited licence by giving you a personalised account to access and use the contents and
-resources of the Porsche Design System only for creating software, applications, websites, interfaces and any other
-products on behalf of Porsche AG or a subsidiary. You may not use the Porsche Design System or any of its contents
-including trade names, trademarks, service marks or product names of Porsche AG, without the expressly permission of
-Porsche AG. It is forbidden to copy and/or reuse contents of the Porsche Design System without an authorised access
-account. You may not reproduce and distribute copies of the Porsche Design System in any medium, with or without
-modification, that is not for Porsche AG or a legal entity.
-
-## Ownership
-
-You agree that Porsche AG owns all legal rights and titles and interests regarding the Porsche Design System and its
-contents, including any intellectual property rights subsisting in Porsche AG or relating to it. These include any
-rights under patent law, copyright law, trade secret law, and trademark law, as well as any other proprietary rights.
-Porsche AG also reserves all rights not expressly granted to you in conjunction with the Porsche Design System.
-
-## Termination
-
-Porsche AG may terminate this agreement if Porsche AG stops developing and distributing the Porsche Design System. In
-that eventuality, you must also stop using the Porsche Design System and delete all copies of it.
-
-## Warranty
-
-Porsche AG provides the Porsche Design System (and all contributors provide their contributions) on an "as-is" basis,
-without warranties or conditions of any kind, either express or implied, including but not limited to warranties of
-merchantability, and any warranties or conditions of title, non-infringement or fitness for a particular purpose. You
-are solely responsible for determining the appropriateness of using the Porsche Design System and assume any associated
-risks under this licence. In no event shall Porsche AG be liable for any claim, damages or other liability arising from
-or in connection with the use of the Kit.
+<div v-html="license"></div>
 
```

Comment:
Can we make the imported LICENSE file somehow more semantic, e.g. also as LICENSE.MD? The rendered HTML looks a bit weird and without any structure.